.github/ISSUE_TEMPLATE/1_Bug_report.yaml

```diff
@@ -0,0 +1,41 @@
+name: 🐛 Bug Report
+description: ⚠️ NEVER report security issues
+labels: [bug]
+
+body:
+  - type: input
+    id: affected-versions
+    attributes:
+      label: Fakturoid PHP lib version(s) affected
+      placeholder: x.y.z
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A clear and concise description of the problem
+    validations:
+      required: true
+  - type: textarea
+    id: how-to-reproduce
+    attributes:
+      label: How to reproduce
+      description: |
+        ⚠️  This is the most important part of the report ⚠️
+        Without a way to easily reproduce your issue, there is little chance we will be able to help you and work on a fix.
+        Please, take the time to show us some code and/or config that is needed for others to reproduce the problem easily.
+    validations:
+      required: true
+  - type: textarea
+    id: possible-solution
+    attributes:
+      label: Possible Solution
+      description: |
+        Optional: only if you have suggestions on a fix/reason for the bug
+        Don't hesitate to create a pull request with your solution, it helps get faster feedback.
```

Comment:
I'd add something like this "… feedback. Please make pull requests as small as possible, don't mix various changes (e.g. adding a feature _and_ fixing code style in a lot of places), as it makes review much harder and we will most likely want you to separate the changes."